server/http/src/service.rs

```diff
@@ -143,8 +167,9 @@ mod tests {
                  ",
             ).unwrap(),
         })));
-        let (query_sink, _) = channel(1);
-        let mut service = GraphQLService::new(schema, query_sink);
+
+        let query_runner = Arc::new(TestQueryRunner::default());
```

Comment:
You don't need the `Default` derive, you can write simply `TestQueryRunner` and that will make one for you.